content/guides/models/registry/create_registry.md

```diff
@@ -7,23 +7,71 @@ title: Create a custom registry
 weight: 2
 ---
 
-Create a [custom registry]({{< relref "./registry_types.md#custom-registry" >}}) for each step of your ML workflow.
+A custom registry offers flexibility and control over the artifact types that you can use, the visibility of the registry to other teams, and more.
 
-Custom registries are particularly useful for organizing project-specific requirements that differ from the default, [core registry]({{< relref "./registry_types.md#core-registry" >}}).
+{{% pageinfo color="info" %}}
+See the summary table in [Registry types]({{< relref "registry_types.md#summary" >}}) for a complete comparison of core and custom registries.
+{{% /pageinfo %}}
 
-The following procedure describes how to interactively create a registry:
-1. Navigate to the **Registry** App in the W&B App UI.
+
+## Create a custom registry
+
+The following procedure describes how to create a custom registry:
+1. Navigate to the **Registry** App at https://wandb.ai/registry/.
 2. Within **Custom registry**, click on the **Create registry** button.
 3. Provide a name for your registry in the **Name** field.
 4. Optionally provide a description about the registry.
 5. Select who can view the registry from the **Registry visibility** dropdown. See [Registry visibility types]({{< relref "./configure_registry.md#registry-visibility-types" >}}) for more information on registry visibility options.
 6. Select either **All types** or **Specify types** from the **Accepted artifacts type** dropdown.
 7. (If you select **Specify types**) Add one or more artifact types that your registry accepts.
+8. Click on the **Create registry** button. 
+
 {{% alert %}}
-An artifact type can not be removed from a registry once it is added and saved in the registry's settings.
+An artifact type cannot be removed from a registry once it is saved in the registry's settings.
 {{% /alert %}}
-8. Click on the **Create registry** button. 
 
 {{< img src="/images/registry/create_registry.gif" alt="" >}}
 
-For example, the preceding image shows a custom registry called "Fine_Tuned_Models" that a user is about to create. The registry is set to **Restricted** which means that only members that are manually added to the "Fine_Tuned_Models" registry will have access to this registry.
\ No newline at end of file
+For example, the preceding image shows a custom registry called `Fine_Tuned_Models` that a user is about to create. The registry is **Restricted** to only members that are manually added to the registry.
+
+
+## Visibility types
+
+The *visibility* of a registry determines who can access that registry. Restricting the visibility of a custom registry helps ensure that only specified members can access that registry.
```

Comment:
```suggestion
The *visibility* of a registry determines who can access that registry. You can restrict the visibility of a custom registry so that only specified members can access that registry.
```